src/Microsoft.Extensions.Logging.Testing/Xunit/LoggedTestInvoker.cs

```diff
@@ -51,6 +51,11 @@ protected override async Task<decimal> InvokeTestMethodAsync(object testClassIns
             {
                 if (i > 0)
                 {
+                    if (!loggedTestBase.RetryPredicate(Aggregator.ToException()))
```

Comment:
loggedTestBase does not guarantee RetryPredicate is set

Reply:
Actually it does. The RetryTestAttribute guarantees that the predicated is set. Then if a RetryTestAttribute can be resolved, the loggedTestBase sets both TestRetries and RetryPredicate. Since this code path can only be triggered if TestRetries is greater than 1, RetryPredicate is guaranteed to be set.